cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -351,14 +360,16 @@ private String rebalanceStateConditionType(KafkaRebalanceStatus status) {
         RebalanceOptions.RebalanceOptionsBuilder rebalanceOptionsBuilder = convertRebalanceSpecToRebalanceOptions(kafkaRebalance.getSpec());
 
         return computeNextStatus(reconciliation, host, apiClient, kafkaRebalance, currentState, rebalanceAnnotation, rebalanceOptionsBuilder)
-           .compose(desiredStatus -> {
+           .compose(desiredStatusandMap -> {
```

Comment:
I guess this should be `desiredStatusAndMap` with capital `A`?

Reply:
I have made these changes. Pushing it to the branch now

Reply:
This may be an issue at my end (I have refreshed the page several times) but I am not seeing this change to `desiredStatusAndMap`. Are you sure you have pushed your latest changes?